packages/fxa-auth-server/lib/luaScripts/getAccessTokens_1.lua

```diff
@@ -1,14 +1,18 @@
 local userId = KEYS[1]
-local tokenIds = redis.call('smembers', userId)
-local result = {}
-if tokenIds then
-  for _, tokenId in ipairs(tokenIds) do
-    local t = redis.call('get', tokenId)
-    if t then
-      table.insert(result, t)
+local ids = redis.call('smembers', userId)
+local results = {}
+if #ids > 0 then
```

Comment:
To check my understanding, this returns all access tokens for a userId. If there is a token stored in the set returned but doesn't have a record in redis, it gets deleted from the token set?

Reply:
correct